recipe/meta.yaml

```
@@ -12,28 +13,26 @@ source:
     - setup.py.patch
 
 build:
-  number: 1
-  skip: true  # [not linux]
-  skip: true  # [cuda_compiler_version == "None"]
-  skip: true  # [cuda_compiler_version == "10.2"]
-  skip: true  # [cuda_compiler_version == "11.0"]
+  number: 2
+  skip: true  # [(not linux) or cuda_compiler_version in (undefined, "None")]
+  ignore_run_exports_from:
+    - {{ compiler("cuda") }}
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   build:
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
-    - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
-    - setuptools
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
```

Looks like this was missed

```suggestion
    - {{ compiler("cxx") }}
    - {{ compiler("cuda") }}
```

---

> - {{ compiler("cuda") }}

Build is failing with this.

---

That will be fixed by re-rendering, but we need to include the change first for that to work